common/components/controllers/AboutUserController.jsx

```diff
@@ -49,35 +49,19 @@ class AboutUserController extends React.PureComponent<{||}, State> {
         title={user && user.first_name + " " + user.last_name + " | DemocracyLab"}
         />
       <div className="AboutProjectController-root">
-        <div className="container-fluid">
-          <div className="background-light">
-            <div className="row" style={{margin: "30px 0 0 0", padding: "10px 0"}}>
-              <div className="col-sm-5">
-                <div className="row">
-                  <div className="col-sm-auto">
-                    <Avatar user={user} size={50} />
-                  </div>
-                  <div className="col">
-                    <div className="row">
-                      <div className="col">
-                        {user && user.first_name + " " + user.last_name}
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
-              <div className="col">
-              </div>
-              <div className="col col-sm-3">
-              </div>
+        <div className="container">
```

Comment:
As you can see from the simplification across this entire file I'm letting Bootstrap do more of our style lifting on this page. It's a change from how it used to display but not a redesign per se; I look at it more like falling in line with the rest of the site. 

Still, because it does involve a UI change, it should get some special attention on review to make sure we're all okay with it until the profile page gets to the top of design's queue.